applications/OptimizationApplication/python_scripts/output/container_variable_data_holder_output_decorator.py

```diff
@@ -0,0 +1,228 @@
+from abc import ABC
+from abc import abstractmethod
+
+import KratosMultiphysics as Kratos
```

Comment:
Need to have a different VTKOutput mechanism for `ContainerVariableData`. Need to add the same for `HDF5`.